Makefile

```diff
@@ -1,6 +1,14 @@
+PKGS     = $(or $(PKG),$(shell env GO111MODULE=on $(GO) list ./...))
+TESTPKGS = $(shell env GO111MODULE=on $(GO) list -f \
+            '{{ if or .TestGoFiles .XTestGoFiles }}{{ .ImportPath }}{{ end }}' \
+            $(PKGS))
+GO 	= go
```

Comment:
Great idea! I have three comments

1. Rename `GO` to `GO_COMPILER`, `GC`, `GO_BIN`, or anything more descriptive
2. Add a comment, something like `# GC is the go compiler/binary for compiling Go data.`
3. Remove the white space before the `=`.

Reply:
will do :)